src/test/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardConsumerTest.java

```diff
@@ -657,7 +658,7 @@ public final void testConsumeShardWithTransientTerminateError() throws Exception
         }
         assertThat(consumer.getCurrentState(), equalTo(ConsumerStates.ShardConsumerState.SHUTDOWN_COMPLETE));
 
-        assertThat(processor.getShutdownReason(), is(equalTo(ShutdownReason.ZOMBIE)));
+        assertThat(processor.getShutdownReason(), is(equalTo(ShutdownReason.TERMINATE)));
```

Comment:
Looks like this test is supposed to test that ZOMBIE shutdowns due to transient failures get retried. Is this still testing the same thing?

Reply:
Nope, this test was designed to test TERMINATE shut down. But in our old code, we modify that to be ZOMBIE during the shard closure validation. Since this logic has been removed, the shutdown reason should stay unchanged.